modules/openideal_user_registration/openideal_user_registration.info.yml

```diff
@@ -0,0 +1,9 @@
+name: openideal_user_registration
```

Comment:
I think it's too specific for now. Wouldn't it be better to change it to openideal_user?